src/test/run-fail/divide-by-zero.rs

```diff
@@ -1,5 +1,5 @@
 // error-pattern:attempt to divide by zero
-
+#[allow(unconditional_panic)]
```

Comment:
@oli-obk This is fine that we now emit this extra lint right?

Reply:
It's there because that test and others were panicking at compile time after the new optimization. The compiler was able to deduce the panic before running the programs :D

Reply:
If I understood that directive correctly, it basically says "hey rustc, it's fine, I know this code will panic. Leave it as-is, please. I wanna see it implode."

Reply:
:rofl: it's fine (ok, no, it's absolutely great!), but please also copy all three of these tests into a single `ui` test so we show how the warning looks.

Maybe also copy these tests to mir-opt and show the diff on const prop (you can merge these into one mir-opt test by having each of the tests in a separate funciton)

Reply:
> but please also copy all three of these tests into a single ui test so we show how the warning looks.

Just so I know I got you correctly: you mean a test that lets the compiler complain about these failures, right? So like, the same as these but without the `#[allow(unconditional_panic)] directive :)

> Maybe also copy these tests to mir-opt and show the diff on const prop

YAS. :rainbow: 

Speaking of mir-opt tests, I'm planning on making a test (maybe test**s**) that showcases where we're at regarding const-prop, with comments saying what the results should ideally be if const-prop was complete.

Reply:
> Maybe also copy these tests to mir-opt and show the diff on const prop

I just did this and, oh boi. Seeing how the `assert!`s that check for invalid operations, in MIR, get a `false` const right where the assertion goes, gives a certain satisfaction that is hard to describe :heart: